src/BrainPass/BrainPass.sol

```diff
@@ -333,6 +321,14 @@ contract BrainPassCollectibles is ERC721, ERC721Pausable, Ownable {
             durationInDays <= MINT_UPPER_LIMIT;
     }
 
+    function _beforeTokenTransfer(
+        address from,
+        address to,
+        uint256 amount
+    ) internal virtual override(ERC721, ERC721Pausable) {
+        super._beforeTokenTransfer(from, to, amount);
```

Comment:
is this really needed if just call super ?

Reply:
its needed to override the _beforeTokenTransfer func in both contracts so this